### DISCUSSION TITLE
what is the up variation of the arrow used for?

### DISCUSSION THREAD

**Discussion by loglot** [TodePond/CellPond#307] [Help] [answered]
all i can see is that it makes a hole in the square for the rule, but nothing else, i am pretty sure that the down arrow is currently unused

**Comment by VINDI777** [comment]
maybe watch the [🤖 Nothing](https://www.youtube.com/watch?v=sQYUQNozljo) video? i think he says something like "with a stamp you can copy the values of an element to another one, and using a diamond*, you can change them"
_*_ note: since the diamond was removed, you use the hexagon now, left side subtraction, right side addition, but also it is very random now. also cant copy from other color channels anymore :(

**Comment by TodePond** [comment, marked as answer]
This comment describes a bit more about squares with holes in: https://github.com/TodePond/CellPond/issues/197#issuecomment-1132831980
